src/lib/tests/e2e.js

```diff
@@ -21,7 +21,8 @@ var dashboard = ParseDashboard({
       "masterKey": "masterKey",
       "appName": "MyApp"
     }
-  ]
+  ],
+  "mountPath": "/dashboard"
 });
 
 app.use('/dashboard', dashboard);
```

Comment:
@flovilmart Can you confirm that `mountPath` should be set when mounting to a non-root path? Previously the `mountPath` was determined in the `getMount` function.